src-web/components/kappnav/modals/ActionModal.js

```diff
@@ -232,7 +243,21 @@ class ActionModal extends React.PureComponent {
                         invalidText={` ${msgs.get('formerror.required')}`} />
                     </FieldWrapper>
                   )
-                } else {
+                }
+                else if (field.type == "multiselect") {
+                  return (
+                    <FieldWrapper key={'fw' + key} labelText={field.label} content={field.description} required={field.optional == false}>
+                      <MultiSelect
+                        id={field.name}
+                        label={field.label}
+                        items = {this.createFiltersForMultiSelectItems(field.values)}
+                        itemToString={item => (item ? item.text : '')}
+                        onChange={(event) => {this.updateSelectedValues(event.selectedItems)}}
```

Comment:
The onChange should invoke the onChange so that the value gets set into the form.  The value that needs to be set is a string of the selected values.  "a,b,c".